src/main/java/org/opentripplanner/graph_builder/module/geometry/GeometryAndBlockProcessor.java

```diff
@@ -646,7 +643,10 @@ private LineString getLineStringForShapeId(FeedScopedId shapeId) {
         CoordinateSequence sequence = new PackedCoordinateSequence.Double(coordinates, 2);
         geometry = geometryFactory.createLineString(sequence);
         geometriesByShapeId.put(shapeId, geometry);
-        distancesByShapeId.put(shapeId, distances);
+
+        if(distances != null) {
+            distancesByShapeId.put(shapeId, distances);
+        }
```

Comment:
```suggestion
        // If we don't have distances here, we can't calculate them ourselves because we can't
        // assume the units will match
        if(hasAllDistances) {
            distancesByShapeId.put(shapeId, distances);
        }
```

Reply:
Remove the `if(distances != null)` block.